.storybook/main.ts

```diff
@@ -3,16 +3,22 @@ import type { RuleSetRule } from 'webpack';
 
 const config: StorybookConfig = {
     stories: ['../docs/**/*.@(md|mdx)', '../src/**/*.stories.@(js|jsx|ts|tsx)', '../src/**/*.@(md|mdx)'],
+
     addons: [
         '@storybook/addon-links',
         '@storybook/addon-essentials',
         { name: '@storybook/addon-styling', options: { postCss: true } },
         '@storybook/addon-designs',
+        '@storybook/addon-mdx-gfm',
+        '@storybook/addon-webpack5-compiler-babel',
+        '@chromatic-com/storybook',
```

Comment:
As Ruggero mentioned, let's part this out for now. Would love to have a spike on its benefits as I see that Storybook has visual testing built in now with the 8 upgrade. That's exciting!